magnite-streaming

jakeseawell opened the pull request:
- Updated magnite-streaming.md
 - Updated audience-field-mapping image
 - Added new audience-target-dropdown image

jakeseawell committed:
magnite-streaming

 - Updated magnite-streaming.md
 - Updated audience-field-mapping image
 - Added new audience-target-dropdown image

jakeseawell committed:
magnite-streaming

 - Updated doc wording

jakeseawell committed:
magnite-streaming

 - Updated page title

jakeseawell committed:
magnite-streaming

 - Updated doc

akoelling commented on help/destinations/catalog/advertising/magnite-streaming.md:
```diff
@@ -63,7 +57,7 @@ Refer to the table below for information about the destination export type and f
 | Item             | Type                            | Notes                                                                                                                                                                                                                                                                                                                              |
 |------------------|---------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| Export type      | **[!UICONTROL Segment export]** | You are exporting all members of a segment (audience) with the identifiers (name, phone number, or others) used in the [!DNL Magnite: Real-Time] destination.                                                                                                                                                            |
+| Export type      | **[!UICONTROL Segment export]** | You are exporting all members of a segment (audience) with the identifiers (name, phone number, or others) used in the [!DNL  Streaming: Real-Time] destination.                                                                                                                                                            |
```
we use the term "Magnite Streaming" elsewhere, so I'd either say adjust the others to just be "Streaming" or update this to be "Magnite Streaming"

jakeseawell replied:
Thx, fat fingered it

akoelling commented on help/destinations/catalog/advertising/magnite-streaming.md:
```diff
@@ -158,16 +160,16 @@ Once your audiences have been uploaded, you may validate your audiences have bee
 * Post-ingest, audiences are expected to appear in [!DNL Magnite Streaming] within a few minutes and can be applied to a deal. You can confirm this by looking up the segment ID that was shared during the activation steps in the Adobe Experience Platform.
 
-## Activate the same audiences through the [!DNL Magnite: Batch ]destination
+## Activate the same audiences through the [!DNL  Streaming: Batch ]destination
```
Ditto to above

jakeseawell replied:
Ditto

akoelling commented on help/destinations/catalog/advertising/magnite-streaming.md:
```diff
@@ -1,50 +1,44 @@
 ## Use cases {#use-cases}
 
-To help you better understand how and when you should use the [!DNL Magnite: Real-Time] destination, here is a sample use case that Adobe Experience Platform customers can solve by using this destination.
+To help you better understand how and when you should use the [!DNL  Streaming: Real-Time] destination, here is a sample use case that Adobe Experience Platform customers can solve by using this destination.
```
see below comment

jakeseawell replied:
same

jakeseawell committed:
magnite-streaming

 - Fixed 'magnite-streaming' typos

jakeseawell committed:
magnite-streaming

 - Change 'Magnite Streaming' to 'Magnite'

jakeseawell committed:
magnite-streaming

 - Updated magnite.streaming.md doc
 - Updated the screenshots for audience-field-mapping and audience-target-dropdown
